etc/openqa/openqa.ini

```diff
@@ -159,3 +159,7 @@ blocklist = job_grab job_done
 # executable name in `$PATH` is called with the job ID as first and only
 # parameter corresponding to the `$result`, for example
 #job_done_hook_failed = my-openqa-hook-failed
+
+## Specify minion failed job task name. The failed jobs for blocklisted tasks will not be counted towards the total of failed jobs.
+[influxdb_minion_fail_job]
+#blocklist = influxdb-minion-fail-job-task-name
```

Comment:
I would have structured this a little bit different. My intention with the first review comment was to have an own section for everything InfluxDB related. I would also use past here (fail**ed**). I also shortened the comment a little bit.

```suggestion
# Configuration for InfluxDB routes
[influxdb]
# Specify Minion task names which should never be counted towards the total of number failed Minion jobs.
#ignored_failed_minion_jobs = influxdb-minion-fail-job-task-name
```